shell/composables/useLabeledFormElement.ts

```diff
@@ -91,6 +96,8 @@ export const useLabeledFormElement = (props: LabeledFormElementProps, emit: (eve
       const message = requiredRule(value);
 
       if (!!message) {
+        emit('update:validation', false);
```

Comment:
We might want to consider also updating `shell/mixins/labeled-form-element.ts` to avoid further drift between the two methods code sharing.